frontend/src/views/FirstTimeUserExperienceView.vue

```diff
@@ -137,7 +137,9 @@ body {
 }
 
 .dark .modal {
+  --background-color: var(--colour-neutral-raised);
   --background: url('@/assets/svg/ftue-background-dark.svg');
+  border: 1px solid var(--colour-apmt-primary);
```

Comment:
```suggestion
  border: 0.0625rem solid var(--colour-apmt-primary);
```

I really need to find a pixel to rem plugin for vite.

Reply:
Ah right, sorry